packages/react/src/graphql/graphql.tsx

```diff
@@ -145,5 +171,24 @@ export const graphql = <Props extends any>(
   GraphQLComponent.displayName = name
   GraphQLComponent.query = query
 
+  GraphQLComponent.getInitialProps = async (ctx: { AppTree: React.FC }) => {
+    let initialProps: Record<any, any>
+
+    if (typeof component.getInitialProps === 'function') {
+      initialProps = await component.getInitialProps(ctx)
+    } else {
+      initialProps = {}
+    }
+
+    if (!isClientSide && ctx?.AppTree) {
+      React.createElement(ctx.AppTree, initialProps)
+    }
+
+    // This gqlQueryName is needed to prevent a Next.js
+    // warning due to this function possibly returning an empty object
+
+    return { gqlessQueryName: query.toString(), ...initialProps }
```

Comment:
Change gqlessQueryName -> gqlessQuery: query

Reply:
the toString is called to get the name of the query btw, since those initialProps has to be json serializable, and it was just a workaround to prevent those warning calls